crosswalk/ado/stata.toc

```diff
@@ -0,0 +1,3 @@
+p make_input_template Write an input template
```

Comment:
Did Matt and Jesse write this? If so, we should give them attribution as for xtreg_cg.